test/keycloak-connect-web-spec.js

```diff
@@ -71,15 +71,14 @@ test('Should login with admin credentials', t => {
 });
 
 test('User should be forbidden to access restricted page', t => {
-  t.plan(1);
+  // t.plan is not necessary with blue-tape; if there's t.plan, the test then doesn't wait for the logout
```

Comment:
If this is not necessary, feel free to remove it. Other than that, looks good to me.